mapper/netlist_util.py

```diff
@@ -1098,6 +1128,7 @@ def tile_to_char(t):
         
         # remove mem reg in conn for manual placement
         graph.remove_mem_reg_tree()
+        # graph.generate_tile_conn(app_dir = app_dir)
```

Comment:
Is this commented-out on purpose?

Reply:
@yuchen-mei  does this make sense? I auto-merged things and didn't have issues with this file

Reply:
this commented function is added for debugging purposes and it won't affect apps functionality

Reply:
> this commented function is added for debugging purposes and it won't affect apps functionality but i can delete it if you think it will hurt the maintainability

No, that's fine, keep it. I just wanted to double-check and make sure that was what you wanted.